boards/scdz/ch32v307evt/doc/index.rst

```diff
@@ -0,0 +1,91 @@
+.. zephyr:board:: ch32v307evt
+
+Overview
+********
+
+The `SCDZ`_ CH32V307EVT hardware provides support for QingKe 32-bit RISC-V4F
```

Comment:
How attached are you to the 'evt' part of the name? I ask as WCH make a series that end with evt, but this board is by a different vendor.